week2/project/ecommerce2/src/components/Products.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import Product from './Product';
+
+export default function Products(props) {
+  const [activeProd, setActiveProd] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState();
```

Comment:
you made this state but you don;t use it
✏️need to modify✏️

Reply:
fixed. I think so;)